### DISCUSSION TITLE
OsmAnd+ on Multiple Platforms

### DISCUSSION THREAD

**Discussion by benpratt** [osmandapp/OsmAnd#13089] [Q&A]
If I wanted to use OsmAnd+ on multiple platforms (e.g. Android and Amazon) would I need to purchase OsmAnd+ for each platform? I have an Android phone which I always have with me but would like to use OsmAnd+ on an Amazon Fire tablet as well so I'm curious if I'll need to purchase OsmAnd+ for each device.

**Comment by vshcherb** [comment]
Each platform doesn't have any connections and each platform has own roalty share, so you need to purchase again but it doesn't depend on OsmAnd decision at all
